### DISCUSSION TITLE
Change to API since 6.3

### DISCUSSION THREAD

**Discussion by CFox243** [snipe/snipe-it#15285] [Q&A]
Could you please clarify what changes, if any, occurred between versions 6.3 and 7.0.10? I am currently running a modified version of Marksman, which I understand is not officially supported here. However, I am attempting to resolve an issue where everything was functioning as expected until I updated to version 7.0.10.

Currently, creation processes work as intended, but updates fail to post without returning any errors. I am trying to identify the root cause of this issue. Your assistance would be greatly appreciated.

Thank you for your time.

Edit:
I believe it's a change from v7.0.9 to v7.0.10

**Comment by snipe** [comment]
If you upgrade to latest on master, this will likely be fixed. There was a missing PUT request handler in the routes that was fixed a little while back, but we haven't tagged a new release yet.

**Comment by CFox243** [comment]
after re-pulling docker latest. the issues seems to be resolved. 
Thank you saves me work.
